### ISSUE TITLE
Getting @intlify/vue-i18n/no-missing-key warning.

### ISSUE THREAD

**Issue by ohno-kouki** [intlify/eslint-plugin-vue-i18n#244]
**Tell us about your environment**
- **Vue version: 3.2.20**
- **ESLint version: 6.7.2**
- **eslint-plugin-vue version: 7.20.0**
- **eslint-plugin-vue-i18n version: 0.15.0**
- **Node version: 14.17.0**

**Please show your full configuration:**

```json5
module.exports = {
  root: true,
  env: {
    node: true
  },
  'extends': [
    'plugin:vue/vue3-recommended',
    'eslint:recommended',
    '@vue/typescript/recommended',
    'plugin:@intlify/vue-i18n/recommended',
  ],
  parserOptions: {
    ecmaVersion: 2020
  },
  rules: {
    indent: ['error', 2, { "SwitchCase": 1 }],
    semi: [
      'error',
      'always'
    ],
    '@typescript-eslint/no-empty-function': [
      'error',
      {
        allow: [
          'functions',
          'arrowFunctions',
          'methods'
        ]
      }
    ],
    'no-console': 'off',
    'vue/max-attributes-per-line': 'off',
    'no-multi-spaces': 2,
    'no-multiple-empty-lines': [2, { 'max': 1 }],
    'no-trailing-spaces': 2
  },
  overrides: [
    {
      files: [
        '**/__tests__/*.{j,t}s?(x)',
        '**/tests/unit/**/*.spec.{j,t}s?(x)'
      ],
      env: {
        jest: true
      }
    }
  ]
};
```

**What did you do?**

A single file component used the contents of a message defined in a separate file.

```vue
<script setup lang="ts">
import { useI18n } from 'vue-i18n';
import { languages } from './Languages';

const { t } = useI18n(languages);
</script>

<template>
  <div>
    {{ t('hello') }}
  </div>
</template>
```

```ts
// Languages.ts
export const languages = {
  messages: {
    ja: {
      hello: 'こんにちは'
    },
    en: {
      hello: 'Hello'
    }
  }
};
```

**What did you expect to happen?**
No warnings.

**What actually happened?**

Warning appears in the first line of the vue file.

【Warning message】
You need to set 'localeDir' at 'settings', or '<i18n>' blocks. See the 'eslint-plugin-vue-i18n' documentation
